bundles/mapping/mapmodule/plugin/vectortilelayer/VectorTileLayer.js

```diff
@@ -15,6 +16,7 @@ Oskari.clazz.define('Oskari.mapframework.mapmodule.VectorTileLayer',
         _createEmptyStyle: function () {
             return createDefaultStyle();
         },
+        getClusteringDistance () {},
```

Comment:
There should be some comment about why a getter with no return value is required. Or this should inherit some generic impl from a parent class or something like that. Why is an empty function required?